website/docs/reference/specifying-dependencies.md

```diff
@@ -96,6 +108,14 @@ where `y ≥ z` and `x > 0`.
 It is possible to further tweak the logic for selecting compatible versions using special operators, though it shouldn't
 be necessary most of the time.
 
+By default, Scarb uses the official registry as a package source.
+Therefore, Scarb will look for the package in the official registry if only the `version` key is specified.
+
+```toml
+[dependencies]
+alexandria_math = { version = "0.1.0" }
```

Comment:
which is correct, the above or this?:
```suggestion
alexandria_math = "0.1.0"
```

Reply:
They're equivalent, this explains that mentioned snippet:
```toml
alexandria_math = { version = "0.1.0" }
```
will default to using the official registry if no other keys are specified.

Reply:
It's unclear from the doc that the format is interchangeable

Reply:
Addressed in 62da5d680363497f033ef8816c3717dc04e4679c, please TAL